### PR TITLE
refactor: Pass scheme eval as argument to session.

### PR DESCRIPTION
**Pull request by prmukherj**
Pass scheme eval as an argument to session instead of taking it directly from FluentConnection.

### CODE REVIEW

File: src/ansys/fluent/core/session.py [hunk 1]
```diff
@@ -102,6 +104,9 @@ def __init__(
         ----------
         fluent_connection (:ref:`ref_fluent_connection`):
             Encapsulates a Fluent connection.
+        scheme_eval: SchemeEval
+            Instance of SchemeEval on which Fluent's scheme code can be
+            executed.
```
Comment (PipKat) [inline src/ansys/fluent/core/session.py:109]:
```suggestion
            Instance of ``SchemeEval`` to execute Fluent's scheme code on.
```

File: src/ansys/fluent/core/session.py [hunk 2]
```diff
@@ -102,6 +104,9 @@ def __init__(
...
         file_transfer_service : Optional
             Supports file upload and download.
```
Comment (PipKat) [inline src/ansys/fluent/core/session.py:111]:
```suggestion
            File service for uploading and downloading files.
```

File: src/ansys/fluent/core/session_meshing.py [hunk 3]
```diff
@@ -28,10 +30,17 @@ def __init__(
 
         Args:
             fluent_connection (:ref:`ref_fluent_connection`): Encapsulates a Fluent connection.
+            scheme_eval: SchemeEval
+                Instance of SchemeEval on which Fluent's scheme code can be executed.
```
Comment (PipKat) [inline src/ansys/fluent/core/session_meshing.py:34]:
```suggestion
                Instance of ``SchemeEval`` to execute Fluent's scheme code on.
                The default is ``None``.
```

File: src/ansys/fluent/core/session_meshing.py [hunk 4]
```diff
@@ -28,10 +30,17 @@ def __init__(
 
         Args:
             fluent_connection (:ref:`ref_fluent_connection`): Encapsulates a Fluent connection.
+            scheme_eval: SchemeEval
```
Comment (PipKat) [inline src/ansys/fluent/core/session_meshing.py:33]:
```suggestion
            scheme_eval: SchemeEval, optional
```

File: src/ansys/fluent/core/session_pure_meshing.py [hunk 5]
```diff
@@ -38,10 +40,17 @@ def __init__(
 
         Args:
             fluent_connection (:ref:`ref_fluent_connection`): Encapsulates a Fluent connection.
+            scheme_eval: SchemeEval
+                Instance of SchemeEval on which Fluent's scheme code can be executed.
```
Comment (PipKat) [inline src/ansys/fluent/core/session_pure_meshing.py:44]:
```suggestion
                Instance of ``SchemeEval`` to execute Fluent's scheme code on..
```

File: src/ansys/fluent/core/session_pure_meshing.py [hunk 6]
```diff
@@ -38,10 +40,17 @@ def __init__(
...
             file_transfer_service: Supports file upload and download.
```
Comment (PipKat) [inline src/ansys/fluent/core/session_pure_meshing.py:45]: Same edits here as suggested earlier.

File: src/ansys/fluent/core/session_solver_icing.py [hunk 7]
```diff
@@ -28,10 +30,17 @@ def __init__(
 
         Args:
             fluent_connection (:ref:`ref_fluent_connection`): Encapsulates a Fluent connection.
+            scheme_eval: SchemeEval
+                Instance of SchemeEval on which Fluent's scheme code can be executed.
```
Comment (PipKat) [inline src/ansys/fluent/core/session_solver_icing.py:34]:
```suggestion
                Instance of ``SchemeEval`` to execute Fluent's scheme code on.
```

File: src/ansys/fluent/core/session_solver_lite.py [hunk 8]
```diff
@@ -16,16 +16,24 @@ class SolverLite(Solver):
...
+        scheme_eval=None,
         start_transcript: bool = True,
         launcher_args: Optional[Dict[str, Any]] = None,
     ):
         """SolverLite session.
 
         Args:
             fluent_connection (:ref:`ref_fluent_connection`): Encapsulates a Fluent connection.
+            scheme_eval: SchemeEval
+                Instance of SchemeEval on which Fluent's scheme code can be executed.
```
Comment (PipKat) [inline src/ansys/fluent/core/session_solver_lite.py:28]:
```suggestion
                Instance of ``SchemeEval`` to execute Fluent's scheme code on.
```

**Review by seanpearsonuk — APPROVED** [review]

**Review by hpohekar — APPROVED** [review]

**Review by ajain-work — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Pull request merged**